schedule/yast/sles+sdk+proxy_SCC_via_YaST_Online_pvm.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: sles+sdk+proxy_SCC_via_YaST
+description: >
+  Add add-on via YaST control center directly https://progress.opensuse.org/issues/16402
+vars:
+  DESKTOP: gnome
+  SCC_ADDONS: sdk
+  SCC_REGISTER: yast
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_module_desktop
```

Comment:
we can use here `installation/module_registration/skip_module_registration`
In pVM we don't need desktop (probably the test suite in Full is wrongly configured).

Reply:
> we can use here `installation/module_registration/skip_module_registration` In pVM we don't need desktop (probably the test suite in Full is wrongly configured).

Thanks, there is no problem with the test suite in Full, I register desktop here for this case because the original case ( https://openqa.suse.de/tests/8752091) is registered with desktop. So I would like to double confirm here it is fine to use role of text mode for this case, right?

Reply:
> we can use here `installation/module_registration/skip_module_registration` In pVM we don't need desktop (probably the test suite in Full is wrongly configured).

Thanks, there is no problem with the test suite in Full, I register desktop here for this case because the original case ( https://openqa.suse.de/tests/8752091) is registered with desktop. So I would like to double confirm here it is fine to use role of text mode for this case, right?